apps/etterlatte-beregning/src/main/kotlin/avkorting/etteroppgjoer/EtteroppgjoerService.kt

```diff
@@ -196,6 +201,22 @@ class EtteroppgjoerService(
         }
     }
 
+    private suspend fun hentVedtaksperioderIEtteroppgjoersAar(
+        sakId: SakId,
+        etteroppgjoersAar: Int,
+    ): List<VedtakSamordningPeriode> {
+        val vedtaksliste = vedtakKlient.hentVedtakslisteForEtteroppgjoersAar(sakId, etteroppgjoersAar, HardkodaSystembruker.etteroppgjoer)
+
+        val vedtakPerioder =
+            vedtaksliste
+                .flatMap { it.perioder }
+                .filter { it.fom.year == etteroppgjoersAar || it.tom?.year == etteroppgjoersAar }
+        // TODO: validere riktig perioder innenfor etteroppgjoersAar?
+        // TODO: reparere så vi får riktig?
```

Comment:
Jeg tror ikke vi trenger reparering her for å regne ut riktig, siden fasiten skal være rett fra vedtak